obal/data/modules/rpmspec_changelog_check.py

```diff
@@ -7,23 +7,49 @@
 import glob
 
 ANSIBLE_METADATA = {
-    'metadata_version': '1.1',
+    'metadata_version': '1.2',
     'status': ['preview'],
     'supported_by': 'community'
 }
 
 from ansible.module_utils.basic import AnsibleModule  # pylint: disable=C0413
 
+def get_changelog_evr(specfile):
```

Comment:
pure python version (with the setup like below):

```python
>>> s.sourceHeader[rpm.RPMTAG_CHANGELOGNAME][0].split(" ")[-1]
'1:1.20.0-0.1.develop'
```

Reply:
The question is if you want to rely on rpm as a library. It can have odd python 2 & python 3 implications. Depending on it via standard tooling doesn't work because RPM isn't a real python package.